### ISSUE TITLE
Can register an whatsapp-account without having a smartphone?

### ISSUE THREAD

**Issue by dominikwille** [venomous0x/WhatsAPI#58]
Wouldn't it be nice if there were a possibility to create a whatsapp account with the WhatsAPI? well actually i don't know if this is even possible...

**Comment by chandukrish** [comment]
Yeah we can create whatsapp account without smart phone.But we need laptop or tablet(without sim) and one normal phone ( not smart phone) is enough.We can create account.For more details see this..http://www.searchforsolutionsonline.com/2015/09/how-to-get-whatsapp-for-free-without.html
